matchms/similarity/CosineHungarian.py

```diff
@@ -56,7 +56,11 @@ def get_matching_pairs():
             matching_pairs = collect_peak_pairs(spec1, spec2, self.tolerance, shift=0.0,
                                                 mz_power=self.mz_power,
                                                 intensity_power=self.intensity_power)
-            return sorted(matching_pairs, key=lambda x: x[2], reverse=True)
+            if matching_pairs is not None:
+                matching_pairs = numpy.array(matching_pairs)
+                matching_pairs = matching_pairs[numpy.argsort(matching_pairs[:, 2])[::-1], :]
+                return matching_pairs
+            return None
```

Comment:
Same as in matchms/similarity/CosineGreedy.py can you do `if..return None`.

Reply:
done.